application.go

```diff
@@ -161,18 +168,55 @@ func (r *Application) Memory(memory float64) *Application {
 // Count sets the number of instances of the application to run
 //		count:	the number of instances to run
 func (r *Application) Count(count int) *Application {
-	r.Instances = count
+	r.Instances = &count
 
 	return r
 }
 
-// Arg adds one or more arguments to the applications
+// AddArgs adds one or more arguments to the applications
 //		arguments:	the argument(s) you are adding
-func (r *Application) Arg(arguments ...string) *Application {
+func (r *Application) AddArgs(arguments ...string) *Application {
 	if r.Args == nil {
-		r.Args = make([]string, 0)
+		r.EmptyArgs()
+	}
+
+	args := *r.Args
+	args = append(args, arguments...)
+	r.Args = &args
+
+	return r
+}
+
+// EmptyArgs explicitly empties arguments -- use this if you need to empty
+// arguments of an application that already has arguments set (setting args to nil will
+// keep the current value)
+func (r *Application) EmptyArgs() *Application {
+	empty := []string{}
+	r.Args = &empty
```

Comment:
Can be simplified by inlining the address of the empty value. Specifically,

`r.Args = &[]string{}`